src/components/dependencies/RescoringModal.js

```diff
@@ -427,6 +427,15 @@ const RescoringFilterOption = ({
   const theme = useTheme()
   const [selected, setSelected] = React.useState([])
 
+  React.useEffect(() => {
+    // remote selected filter if option is not available anymore (e.g. last rescoring was deleted)
+    if (selected.every((s) => options.some((o) => o.name === s))) return
```

Comment:
I think this is quite hard to read. How about constructing a set of selected names, and check whether the set of available option-names is a subset instead?

Reply:
oh wow, there is [Set.isSubsetOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/isSubsetOf) 🤯 

> Baseline 2024 Newly available

It appears this feature was released quite recently.

Will update, but for the record:
We will rather have to check whether selected is subset of options, not the other way around.